diff --git a/simulation/PrecRecF1/PrecRecF1.R b/simulation/PrecRecF1/PrecRecF1.R
index 77d43da..a77c0fa 100755
--- a/simulation/PrecRecF1/PrecRecF1.R
+++ b/simulation/PrecRecF1/PrecRecF1.R
@@ -1,4 +1,4 @@
-#!/usr/bin/Rscript
+#!/home/kieran/src/R-3.2.4/bin/Rscript
 require(ggplot2)
 require(dplyr)
 require(RColorBrewer)
diff --git a/simulation/PrecRecF1/PrecRecF1_lethals.R b/simulation/PrecRecF1/PrecRecF1_lethals.R
index b219605..c9165fc 100755
--- a/simulation/PrecRecF1/PrecRecF1_lethals.R
+++ b/simulation/PrecRecF1/PrecRecF1_lethals.R
@@ -1,4 +1,4 @@
-#!/usr/bin/Rscript
+#!/home/kieran/src/R-3.2.4/bin/Rscript
 require(ggplot2)
 require(dplyr)
 require(RColorBrewer)
